packages/repository/src/entity/factory.test.ts

```diff
@@ -31,26 +24,41 @@ describe("Entity", () => {
       id: "super-unique-id",
       foo: "bar",
     })
+
+    vi.clearAllMocks()
   })
 
   describe("Factories", () => {
     describe("Model Factory", () => {
-      it("Given schema, When model factory called, Then return context factory", () => {
-        const result = entityModelFactory<TestRawEntityData>()
+      it("Given schema, relations, sync keys, methods, When entity model factory called, Then return entity factories", ({
+        entitySchema,
+        entityMethods,
+        syncKeys,
+        authorsRelationship,
+      }) => {
```

Comment:
My w sumie w BP zregyznowaliśmy z ustawiana contextu i destrukturyzowania tego w test case'ach. Bo powstawały czasami takie duże twory właśnie. Teraz po prostu inicjalizujemy sobie dane na górze i przekazujemy jako parametr w given odrazu, czyli tutaj by bylo const result = entityModelFactory(obj) i mamy 8 lini mniej. Czy nie możemy tak robić? Jakie plusy niesie za sobą ten context?

Reply:
Myślę, że jak prześledziłbyś pochodzenie tych wartości to odpowiedź nasuneła by się sama ;) Jest jedno źródło tych testowych wartości i nie musimy go powielać w każdym teście.

Linie kodu nigdy nie będą dla mnie wyznacznikiem czy coś jest lepiej zrobione czy nie - to jedynie formatowanie, równie dobrze można to zapisać w jednej linijce nie zmieniając kompletnie niczego w konstrukcji tego testu 😄